scss/_settings_colors.scss

```diff
@@ -1,47 +1,52 @@
+@import '../build/scss/tokens';
+
 // Global color settings
-$color-transparent: transparent !default;
+$color-transparent: $vf-color-transparent !default;
 
 // Generic gray values (used in elements that have not been themed yet)
-$color-x-light: #fff !default;
-$color-light: #f7f7f7 !default;
-$color-mid-x-light: #e5e5e5 !default;
-$color-mid-light: #d9d9d9 !default;
-$color-mid: #999 !default;
-$color-mid-dark: #666 !default;
-$color-dark: #111 !default;
-$color-x-dark: #000 !default;
-$color-input-shadow: rgba($color-x-dark, 0.12) !default;
+$color-x-light: $vf-color-x-light !default;
+$color-light: $vf-color-light !default;
+$color-mid-x-light: $vf-color-mid-x-light !default;
+$color-mid-light: $vf-color-mid-light !default;
+$color-mid: $vf-color-mid !default;
+$color-mid-dark: $vf-color-mid-dark !default;
+$color-dark: $vf-color-dark !default;
+$color-x-dark: $vf-color-x-dark !default;
+// TODO: This is unused
+$color-input-shadow: $vf-color-input-shadow !default;
```

Comment:
To remove a variable we would need to make sure no one else is using it as well, so it's a bit of a breaking change territory.

But good to identify them. We can either just comment on them that they are deprecated, or maybe move them all to a separate `_settings_deprecated.scss` or something,  where we keep all unused values to be removed in next major release.